pytest_rally/plugin.py

```diff
@@ -62,9 +62,9 @@ def current_branch(repo):
         branches = run_command_with_output(cmd).split("\n")
         current = next(filter(lambda b: b.startswith("*"), branches))
         if "detached" in current:
-            return run_command_with_output(f'git -C {repo} rev-parse HEAD')
+            return run_command_with_output(f'git -C {repo} rev-parse HEAD').strip()
         else:
-            return current.split()[1]
+            return current.split()[1].strip()
```

Comment:
This `strip()` is not needed as branches was split over `"\n"` already. But I'm happy to be over-resilient here.